package.json

```diff
@@ -46,7 +46,7 @@
     "prettier": "^2.0.5",
     "rimraf": "^3.0.0",
     "rollup": "^1.27.5",
-    "rollup-plugin-svelte": "^5.0.3",
+    "rollup-plugin-svelte": "^6.1.1",
```

Comment:
Bumped the plugin as it resolves an issue I was having when compiling. See [git issue](https://github.com/Samuel-Martineau/generator-svelte/issues/7) for more details.